<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>TestAudioSettingDialog</class>
 <widget class="QDialog" name="TestAudioSettingDialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>756</width>
    <height>337</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>音频设置</string>
  </property>
  <property name="statusTip">
   <string>进入房间-&gt;调整-&gt;验证效果</string>
  </property>
  <layout class="QGridLayout" name="gridLayout">
   <property name="sizeConstraint">
    <enum>QLayout::SetDefaultConstraint</enum>
   </property>
   <item row="1" column="0" colspan="2">
    <widget class="QLabel" name="label_4">
     <property name="font">
      <font>
       <pointsize>14</pointsize>
       <weight>75</weight>
       <italic>true</italic>
       <bold>true</bold>
      </font>
     </property>
     <property name="text">
      <string>用户音量设置：</string>
     </property>
    </widget>
   </item>
   <item row="1" column="3" colspan="2">
    <widget class="QLabel" name="label_18">
     <property name="font">
      <font>
       <pointsize>14</pointsize>
       <weight>75</weight>
       <italic>true</italic>
       <bold>true</bold>
      </font>
     </property>
     <property name="text">
      <string>SDK采集音量设置：</string>
     </property>
    </widget>
   </item>
   <item row="2" column="0" colspan="3">
    <widget class="Line" name="line">
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
    </widget>
   </item>
   <item row="2" column="3" colspan="5">
    <widget class="Line" name="line_6">
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
    </widget>
   </item>
   <item row="3" column="0" rowspan="2">
    <widget class="QLabel" name="label_2">
     <property name="font">
      <font>
       <pointsize>14</pointsize>
       <weight>50</weight>
       <bold>false</bold>
      </font>
     </property>
     <property name="text">
      <string>选择用户</string>
     </property>
     <property name="alignment">
      <set>Qt::AlignLeading|Qt::AlignLeft|Qt::AlignVCenter</set>
     </property>
    </widget>
   </item>
   <item row="3" column="2" rowspan="2">
    <widget class="QComboBox" name="comboBoxRemoteUsers">
     <property name="font">
      <font>
       <pointsize>14</pointsize>
      </font>
     </property>
     <property name="styleSheet">
      <string notr="true"/>
     </property>
     <property name="currentText">
      <string notr="true"/>
     </property>
    </widget>
   </item>
   <item row="3" column="3" rowspan="2">
    <widget class="QLabel" name="label_17">
     <property name="font">
      <font>
       <pointsize>14</pointsize>
       <weight>50</weight>
       <bold>false</bold>
      </font>
     </property>
     <property name="text">
      <string>音量设置</string>
     </property>
    </widget>
   </item>
   <item row="4" column="7">
    <widget class="QSlider" name="horizontalSliderAudioCaptureVolume">
     <property name="font">
      <font>
       <pointsize>14</pointsize>
      </font>
     </property>
     <property name="maximum">
      <number>100</number>
     </property>
     <property name="value">
      <number>100</number>
     </property>
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
    </widget>
   </item>
   <item row="5" column="0">
    <widget class="QLabel" name="label_3">
     <property name="font">
      <font>
       <pointsize>14</pointsize>
       <weight>50</weight>
       <bold>false</bold>
      </font>
     </property>
     <property name="text">
      <string>音量设置</string>
     </property>
    </widget>
   </item>
   <item row="5" column="2">
    <widget class="QSlider" name="horizontalSliderRemoteUserVolume">
     <property name="font">
      <font>
       <pointsize>14</pointsize>
      </font>
     </property>
     <property name="maximum">
      <number>100</number>
     </property>
     <property name="value">
      <number>100</number>
     </property>
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
    </widget>
   </item>
   <item row="6" column="0" colspan="2">
    <widget class="QLabel" name="label_5">
     <property name="font">
      <font>
       <pointsize>14</pointsize>
       <weight>75</weight>
       <italic>true</italic>
       <bold>true</bold>
      </font>
     </property>
     <property name="text">
      <string>SDK播放音量设置：</string>
     </property>
    </widget>
   </item>
   <item row="6" column="3" colspan="5">
    <widget class="QLabel" name="label_20">
     <property name="font">
      <font>
       <pointsize>14</pointsize>
       <weight>75</weight>
       <italic>true</italic>
       <bold>true</bold>
      </font>
     </property>
     <property name="text">
      <string>系统声音采集音量设置：</string>
     </property>
    </widget>
   </item>
   <item row="7" column="0" colspan="3">
    <widget class="Line" name="line_2">
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
    </widget>
   </item>
   <item row="7" column="3" colspan="5">
    <widget class="Line" name="line_8">
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
    </widget>
   </item>
   <item row="8" column="0" rowspan="2">
    <widget class="QLabel" name="label_10">
     <property name="font">
      <font>
       <pointsize>14</pointsize>
       <weight>50</weight>
       <bold>false</bold>
      </font>
     </property>
     <property name="text">
      <string>音量设置</string>
     </property>
    </widget>
   </item>
   <item row="8" column="3" rowspan="2">
    <widget class="QLabel" name="label_22">
     <property name="font">
      <font>
       <pointsize>14</pointsize>
       <weight>50</weight>
       <bold>false</bold>
      </font>
     </property>
     <property name="text">
      <string>音量设置</string>
     </property>
    </widget>
   </item>
   <item row="9" column="2">
    <widget class="QSlider" name="horizontalSliderAudioPlayoutVolume">
     <property name="font">
      <font>
       <pointsize>14</pointsize>
      </font>
     </property>
     <property name="maximum">
      <number>100</number>
     </property>
     <property name="value">
      <number>100</number>
     </property>
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
    </widget>
   </item>
   <item row="9" column="7">
    <widget class="QSlider" name="horizontalSliderSystemAudioLoopbackVolume">
     <property name="font">
      <font>
       <pointsize>14</pointsize>
      </font>
     </property>
     <property name="maximum">
      <number>100</number>
     </property>
     <property name="value">
      <number>100</number>
     </property>
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
    </widget>
   </item>
   <item row="10" column="7">
    <widget class="QCheckBox" name="checkBoxSytemAudioLoopbak">
     <property name="enabled">
      <bool>true</bool>
     </property>
     <property name="font">
      <font>
       <pointsize>14</pointsize>
      </font>
     </property>
     <property name="text">
      <string>开启/关闭</string>
     </property>
     <property name="checked">
      <bool>false</bool>
     </property>
    </widget>
   </item>
   <item row="11" column="0" colspan="2">
    <widget class="QLabel" name="label_6">
     <property name="font">
      <font>
       <pointsize>14</pointsize>
       <weight>75</weight>
       <italic>true</italic>
       <bold>true</bold>
      </font>
     </property>
     <property name="text">
      <string>系统合成器音量设置：</string>
     </property>
    </widget>
   </item>
   <item row="11" column="3" colspan="2">
    <widget class="QLabel" name="label_7">
     <property name="font">
      <font>
       <pointsize>14</pointsize>
       <weight>75</weight>
       <italic>true</italic>
       <bold>true</bold>
      </font>
     </property>
     <property name="text">
      <string>当前音频设备设置：</string>
     </property>
    </widget>
   </item>
   <item row="12" column="0" colspan="3">
    <widget class="Line" name="line_3">
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
    </widget>
   </item>
   <item row="12" column="3" rowspan="2" colspan="5">
    <widget class="Line" name="line_4">
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
    </widget>
   </item>
   <item row="13" column="0" rowspan="2">
    <widget class="QLabel" name="label_15">
     <property name="font">
      <font>
       <pointsize>14</pointsize>
       <weight>50</weight>
       <bold>false</bold>
      </font>
     </property>
     <property name="text">
      <string>音量设置</string>
     </property>
    </widget>
   </item>
   <item row="13" column="2" rowspan="2">
    <widget class="QSlider" name="horizontalSliderApplicationVolume">
     <property name="font">
      <font>
       <pointsize>14</pointsize>
      </font>
     </property>
     <property name="maximum">
      <number>100</number>
     </property>
     <property name="value">
      <number>100</number>
     </property>
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
    </widget>
   </item>
   <item row="14" column="3">
    <widget class="QLabel" name="label_16">
     <property name="font">
      <font>
       <pointsize>14</pointsize>
       <weight>50</weight>
       <bold>false</bold>
      </font>
     </property>
     <property name="text">
      <string>音量设置</string>
     </property>
    </widget>
   </item>
   <item row="14" column="6" colspan="2">
    <widget class="QSlider" name="horizontalSliderCurrentDeviceVolume">
     <property name="maximum">
      <number>150</number>
     </property>
     <property name="value">
      <number>100</number>
     </property>
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
    </widget>
   </item>
   <item row="15" column="2">
    <widget class="QCheckBox" name="checkBoxApplicationMute">
     <property name="enabled">
      <bool>true</bool>
     </property>
     <property name="font">
      <font>
       <pointsize>14</pointsize>
      </font>
     </property>
     <property name="text">
      <string>静音</string>
     </property>
     <property name="checked">
      <bool>false</bool>
     </property>
    </widget>
   </item>
   <item row="15" column="5" colspan="3">
    <widget class="QCheckBox" name="checkBoxCurrentDeviceMute">
     <property name="enabled">
      <bool>true</bool>
     </property>
     <property name="font">
      <font>
       <pointsize>14</pointsize>
      </font>
     </property>
     <property name="text">
      <string>静音</string>
     </property>
     <property name="checked">
      <bool>false</bool>
     </property>
    </widget>
   </item>
   <item row="16" column="2">
    <spacer name="verticalSpacer">
     <property name="orientation">
      <enum>Qt::Vertical</enum>
     </property>
     <property name="sizeHint" stdset="0">
      <size>
       <width>219</width>
       <height>2</height>
      </size>
     </property>
    </spacer>
   </item>
   <item row="0" column="3" colspan="5">
    <widget class="QLabel" name="label_11">
     <property name="font">
      <font>
       <pointsize>17</pointsize>
       <weight>75</weight>
       <bold>true</bold>
       <underline>false</underline>
      </font>
     </property>
     <property name="text">
      <string>上行音频设置</string>
     </property>
     <property name="alignment">
      <set>Qt::AlignCenter</set>
     </property>
    </widget>
   </item>
   <item row="0" column="0" colspan="3">
    <widget class="QLabel" name="label">
     <property name="font">
      <font>
       <pointsize>17</pointsize>
       <weight>75</weight>
       <bold>true</bold>
       <underline>false</underline>
      </font>
     </property>
     <property name="text">
      <string>下行音频设置</string>
     </property>
     <property name="alignment">
      <set>Qt::AlignCenter</set>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
